### ISSUE TITLE
GoMatrixHosting no longer a thing

### ISSUE THREAD

**Issue by dzg** [spantaleev/matrix-docker-ansible-deploy#1654]
According to [https://gomatrixhosting.com/2022/02/07/goodbye-gomatrixhosting/](https://gomatrixhosting.com/2022/02/07/goodbye-gomatrixhosting/) I understand that service is gone.

Might want to update https://github.com/spantaleev/matrix-docker-ansible-deploy#services-by-the-community

**Comment by PC-Admin** [comment]
The GoMatrixHosting system is still being developed, although commercial activity has ceased. I would suggest this edit for it:

```
- [GoMatrixHosting](https://gitlab.com/GoMatrixHosting) - matrix-docker-ansible-deploy "as a service" with [Ansible AWX](https://github.com/ansible/awx). Members can be assigned a server from DigitalOcean, or they can connect their on-premises server. This AWX system can manage the updates, configuration, import and export, backups, and monitoring on its own. For more information come [visit us on Matrix](https://matrix.to/#/#general:gomatrixhosting.com).
```

**Comment by PC-Admin** [comment]
Alternatively cut the section out i suppose, as it's not really an available service. :man_shrugging:

**Comment by spantaleev** [comment]
Fixed by #1663
